Contuity/Views/Explore/ExploreViewController.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  ExploreViewController.swift
+//  Contuity
+//
+//  Created by Anthony on 10/15/18.
+//  Copyright © 2018 Generate. All rights reserved.
+//
+import UIKit
+
+// View Protocol for the Explore View
+protocol ExploreViewProtocol: class {}
+
+// View Controller for the Explore View
+class ExploreViewController: UITableViewController {
+    var jots: [Jot] = []
+
+    private (set) var presenter = ExplorePresenter()
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        presenter.attachView(self)
+        prettify()
+        let nib = UINib(nibName: "JotTableViewCell", bundle: nil)
```

Comment:
An easier way to construct nibs is something like JotTableViewCell.nib() as a function call (really minor and probably not worth changing)